HLTrigger/Configuration/python/Phase2/hltPhase2Setup_cff.py

```diff
@@ -0,0 +1,9996 @@
+import FWCore.ParameterSet.Config as cms
+
+#ES Sources
+BTagRecord = cms.ESSource("EmptyESSource",
+    firstValid = cms.vuint32(1),
+    iovIsRunNotTime = cms.bool(True),
+    recordName = cms.string('JetTagComputerRecord')
+)
+
+CSCChannelMapperESSource = cms.ESSource("EmptyESSource",
+    firstValid = cms.vuint32(1),
+    iovIsRunNotTime = cms.bool(True),
+    recordName = cms.string('CSCChannelMapperRecord')
+)
+
+CSCIndexerESSource = cms.ESSource("EmptyESSource",
+    firstValid = cms.vuint32(1),
+    iovIsRunNotTime = cms.bool(True),
+    recordName = cms.string('CSCIndexerRecord')
+)
+
+GlobalTag = cms.ESSource("PoolDBESSource",
+    DBParameters = cms.PSet(
+        authenticationPath = cms.untracked.string(''),
+        authenticationSystem = cms.untracked.int32(0),
+        messageLevel = cms.untracked.int32(0),
+        security = cms.untracked.string('')
+    ),
+    DumpStat = cms.untracked.bool(False),
+    ReconnectEachRun = cms.untracked.bool(False),
+    RefreshAlways = cms.untracked.bool(False),
+    RefreshEachRun = cms.untracked.bool(False),
+    RefreshOpenIOVs = cms.untracked.bool(False),
+    connect = cms.string('frontier://FrontierProd/CMS_CONDITIONS'),
+    globaltag = cms.string('110X_mcRun4_realistic_v3'),
+    pfnPostfix = cms.untracked.string(''),
+    pfnPrefix = cms.untracked.string(''),
+    snapshotTime = cms.string(''),
+    toGet = cms.VPSet(
```

Comment:
instead of carrying around all the customization of tags needed for the RelVal production, if this is considered to be fixed, why not using the physical Global Tag `111X_mcRun4_realistic_T15_v1` which is equivalent in tag content?

Diff: https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/111X_mcRun4_realistic_T15_v1/110X_mcRun4_realistic_v3

```bash
$ conddb diff 110X_mcRun4_realistic_v3 111X_mcRun4_realistic_T15_v1
[2020-07-18 11:18:56,117] INFO: Connecting to pro [frontier://PromptProd/CMS_CONDITIONS]
Record                      Label          pro::110X_mcRun4_realistic_v3 Tag                                           pro::111X_mcRun4_realistic_T15_v1 Tag             
--------------------------  -------------  --------------------------------------------------------------------------  -----------------------------------------------   
SiPixelGenErrorDBObjectRcd  -              SiPixelGenErrorDBObject_phase1_38T_2017_rereco_v9                           SiPixelGenErrorDBObject_phase2_T15_v5_mc          
SiPixelLorentzAngleRcd      -              SiPixelLorentzAngle_phase2_T15_v0_mc                                        SiPixelLorentzAngle_phase2_T15_v5_mc              
SiPixelLorentzAngleRcd      forWidth       SiPixelLorentzAngle_phase2_forWidth_T15_v0_mc                               SiPixelLorentzAngle_phase2_T15_mc_forWidthEmpty   
SiPixelLorentzAngleRcd      fromAlignment  SiPixelLorentzAngle_fromAlignment_v1_mc[cms_orcon_prod/CMS_COND_31X_PIXEL]  SiPixelLorentzAngle_phase2_T15_mc_forWidthEmpty   
SiPixelLorentzAngleSimRcd   -              SiPixelSimLorentzAngle_phase2_T15_v0_mc                                     SiPixelSimLorentzAngle_phase2_T15_v5_mc           
SiPixelTemplateDBObjectRcd  -              SiPixelTemplateDBObject_phase1_38T_2017_rereco_v9                           SiPixelTemplateDBObject_phase2_T15_v5_mc     
```

Reply:
@christopheralanwest FYI

Reply:
If it is the same, I agree that using the GT is the correct approach. I will test a version that uses the GT and checks that the results are identical.

Reply:
Results are identical (as they should be). Will just use `111X_mcRun4_realistic_T15_v1` from now on!

Reply:
Is the content of `111X_mcRun4_realistic_T15_v1` intended to be the default Phase 2 GT or is the use of this GT specific to this PR? I'm wondering if the `auto:phase2_realistic` GT should be updated.

Reply:
@christopheralanwest, I think that GT is the GT supposed to be used in the scope of the HLT TDR studies (at least that's what has been used so far in the validation).
Changing `autoCond` (in 11.1.X only!) to reflect that would certainly be beneficial in my opinion - as I think I pointed out already at the time of closing 11.1.0.